agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -391,6 +389,10 @@ static int run_son_slave(int slave_num, beerocks::config_file::sConfigSlave &bee
     son::slave_thread::sSlaveConfig son_slave_conf;
     fill_son_slave_config(beerocks_slave_conf, son_slave_conf, slave_num);
 
+    // disable stopping on failure initially. Later on, it will be read from BPL as part of
+    // cACTION_PLATFORM_SON_SLAVE_REGISTER_RESPONSE
+    son_slave_conf.stop_on_failure_attempts = 0;
```

Comment:
Since Tomer was anyway giving nitpick comments, I have one too: I think it would be nicer if this were in fill_son_slave_config() instead.

Reply:
Done